css-ui-3/caret-color-016.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color visited link computed value</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
+<link rel="help" href="https://www.w3.org/TR/css3-color/#color0">
+<meta name="flags" content="dom interact">
```

Comment:
This looks like a valid thing to test, so I'd like the test here as well. Just wondering if this is the right way to mark a reftest.js that may need a manual setup. Let’s give a little while to @gsnedders to respond, and if it says yes or doesn't say anything, I'll merge, and if he has some better recommendation, we can follow that.